### DISCUSSION TITLE
Installation Issue at Step 7.4 on MacOS

### DISCUSSION THREAD

**Discussion by DararaiPakraypet** [OpenTOPAS/OpenTOPAS#103] [Run-Problem]
### Contact Details

nipon.sai@edu.cra.ac.th

### What was the issue?

Dear Developer,

I encountered an issue during the installation process outlined in step 7.4 for MacOS. Specifically, after running the command: make -j20 install.

I received an error in the MacOS terminal. Please find the screenshot of the error attached for your reference.

Could you kindly provide guidance on resolving this issue or suggest any alternative steps for successful installation?

Thank you for your support

Best,

<img width="745" alt="1  Step 7 4 problem" src="https://github.com/user-attachments/assets/49383c18-1234-4907-8364-6ffe16b2a4d5">

<img width="785" alt="2  Step 7 4 problem for make" src="https://github.com/user-attachments/assets/b84d25f5-b024-4d00-a4e6-81be3ecc4dd1">

### TOPAS category

Geometry

### Version

4.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi,

There was likely an initial error message further up in your console. Could you please provide the full output after running `make`?

Thanks,

Thongchai

**Reply by DararaiPakraypet** [reply]
Thank you for your response, this is full output after running make.

<img width="885" alt="1" src="https://github.com/user-attachments/assets/930ab6d3-611d-44cf-95a6-1341bf62b096">
<img width="882" alt="2" src="https://github.com/user-attachments/assets/2b5954e9-ae33-4caa-ac25-2be43455eedc">
<img width="881" alt="3" src="https://github.com/user-attachments/assets/d2867d1f-f0b2-48d4-818d-24468f1be68f">
<img width="880" alt="4" src="https://github.com/user-attachments/assets/a7663f49-6b2b-4de3-a34c-b00f12bb307a">
<img width="882" alt="5" src="https://github.com/user-attachments/assets/e3d71499-f655-4a35-b307-5a559c643ede">
<img width="884" alt="6" src="https://github.com/user-attachments/assets/17d2c812-7f5f-4053-973b-c5647e445715">
<img width="881" alt="7" src="https://github.com/user-attachments/assets/d958d4ad-903e-4ab0-95fb-ef3a473ec67c">
<img width="870" alt="8" src="https://github.com/user-attachments/assets/c049699c-3596-4b50-867d-ffea63b66b06">
<img width="884" alt="9" src="https://github.com/user-attachments/assets/00f4c495-b920-4576-b2de-63eb4445c2a2">
<img width="867" alt="10" src="https://github.com/user-attachments/assets/6f91815e-fd1d-469d-b414-af41c0b25f73">
<img width="876" alt="11" src="https://github.com/user-attachments/assets/dcef19b5-f544-45e2-b757-c9b3eca32c60">
<img width="884" alt="12" src="https://github.com/user-attachments/assets/9d7a1640-a615-4a34-9dc1-dd3e873ff536">
<img width="883" alt="13" src="https://github.com/user-attachments/assets/53206065-600e-4fdc-a939-70eef0c6f981">
<img width="882" alt="14" src="https://github.com/user-attachments/assets/56d27f7c-702a-49d4-b15b-86c8b56b4493">
<img width="866" alt="15" src="https://github.com/user-attachments/assets/f7fa3e19-b015-43c7-9d74-2fb9c738c36c">
<img width="879" alt="16" src="https://github.com/user-attachments/assets/98325a9c-38fa-4031-89ca-31ad956c0b68">
<img width="875" alt="17" src="https://github.com/user-attachments/assets/0be1b4db-29e9-4d07-9916-ba64a1d10f3e">

**Comment by tmasilela** [comment]
From what I can see this seems to be a QT issue. TOPAS is only compatible with qt@5, can you verify that `brew list --versions qt@6` doesn't yield any output? Also please verify that you performed step 6.2 correctly (the output of `readlink -f $(brew --prefix qt@5)`) should be used as the path for `-DCMAKE_PREFIX_PATH`

Let me know if any of this solves your issue.

All the best,

Thongchai
